SECURITY.md

```diff
@@ -0,0 +1,64 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+We take the security of Boston Open Source's website and community infrastructure seriously. If you discover a security vulnerability, please follow these steps:
```

Comment:
```suggestion
We take the security of Boston Open Dev's website and community infrastructure seriously. If you discover a security vulnerability, please follow these steps:
```

We've pretty much settled on calling ourselves "Boston Open Dev". Please see https://github.com/orgs/bostonopen/discussions/1